static/js/pivotal.js

```diff
@@ -90,9 +90,13 @@
    * @return {Array}      Array of pivotal IDs
    */
   function getPivotalStoryIDs(msgs) {
-    var storyIDs = msgs.map(function(str) {
-      return parseInt(str.match(/\[#(\d+)\]/)[1], 10);
-    });
+    var storyIDs = msgs
+      .filter(function(str) {
+        return /\[#(\d+)\]/.test(str);
```

Comment:
Can you extract the regex into a variable (or a constant) to avoid the duplication ?